### PR TITLE
Overflow of Caption fixed with added new class and by defining it in …

### PR DESCRIPTION
**Pull request by Shubham185y**
#273  of Caption fixed with added new class and by defining it in css file as well
Before: 
image

After:
image

### CODE REVIEW

File: src/components/Post.jsx [hunk 1]
```diff
@@ -281,7 +281,7 @@ function Post(prop) {
         {postHasImages ? (
           <ImageSlider slides={postImages} isCommentBox={false} />
         ) : (
-          <div className="post__background">{caption}</div>
```
Comment (narayan954) [inline src/components/Post.jsx:284]: it's not an addition if you're replacing the class. We still need the background from post__background class. You may add both classes and verify if it still works

File: src/components/Post.jsx [hunk 2]
```diff
@@ -375,8 +375,8 @@ function Post(prop) {
                     <Item>
                       {postHasImages ? (
                         <ImageSlider slides={postImages} isCommentBox />
-                      ) : (
-                        <div className="post__background">{caption}</div>
```
Comment (narayan954) [inline src/components/Post.jsx:379]: same as earlier comment

File: src/index.css [hunk 3]
```diff
@@ -152,6 +152,11 @@
   position: relative;
 }
 
+.post_caption{
+  width: 384px;
```
Comment (narayan954) [inline src/index.css:156]: try not to hardcode in pixels, use responsive measurement units like em, rem

**Review by narayan954 — CHANGES_REQUESTED** [review]

**Comment by Shubham185y** [comment]
image
responsive as well:
image

**Review by narayan954 — APPROVED** [review]
looks good to me!

**Comment by narayan954** [comment]
Thanks for your contribution :)

**Pull request merged**